validators/types_unix.go

```diff
@@ -40,11 +40,12 @@ var DefaultSysSpec = SysSpec{
 			{Name: "IPC_NS"},
 			{Name: "UTS_NS"},
 			{Name: "CGROUPS"},
-			{Name: "CGROUP_CPUACCT"},
+			{Name: "CGROUP_BPF"},     // cgroups v2
```

Comment:
do you know where does it say that v2 requires CGROUP_BPF?
no info here:
https://docs.kernel.org/admin-guide/cgroup-v2.html

Reply:
`device` controller is based on BPF. Let me check if there is more official information about that.

Reply:
```
Cgroup v2 device controller has no interface files and is implemented
on top of cgroup BPF. 
```
From https://www.kernel.org/doc/Documentation/cgroup-v2.txt.